app/src/main/res/values/colors.xml

```diff
@@ -44,6 +44,9 @@
     <!-- A really bright Holo shade of gray -->
     <color name="gray_bright">#33CCCCCC</color>
 
+    <!-- Redesign Grey -->
+    <color name="grey_redesign">#7A7A7A</color>
```

Comment:
can we use a better `name` for it?
If we are designing the app interface, doesn't mean we will use redesign word in our app's code.

Reply:
Okay. Is `grey_light` a better name? I'll change the comment accordingly.